### PR TITLE
adding tag/remove-tag actions to log-group

### PR DESCRIPTION
**Pull request by joshuaroot**
Per #2240 
- adding tag/remove-tag actions to log-group

### CODE REVIEW

File: c7n/resources/cw.py [hunk 1]
```diff
@@ -304,3 +306,79 @@ def process_resource_set(self, client, accounts, resources):
...
+                    value: DesiredTagValue
+    """
+    permissions = ('logs:TagLogGroup',)
+
+    def process_resource_set(self, resources, tags):
+        client = local_session(self.manager.session_factory).client('logs')
+
+        def _add_tags(r):
+            try:
+                tag_dict = {}
```
Comment (thisisshi) [inline c7n/resources/cw.py]: could simplify this with a comprehension, `tag_dict = {t['key']:t['value'] for t in tags}`

File: tests/test_cwl.py [hunk 2]
```diff
@@ -87,3 +87,41 @@ def test_delete(self):
...
+                'key': 'DesiredTag',
+                'value': 'DesiredValue'}]}, session_factory=session)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['logGroupName'], log_group)
+        self.assertEqual(
+            client.list_tags_log_group(logGroupName=log_group)['tags'],
+            {'DesiredTag': 'DesiredValue'})
+
+    def test_remove_tag(self):
```
Comment (thisisshi) [inline tests/test_cwl.py]: afaics this doesn't actually test removing the tag as the 'DesiredTag' tag is never applied to the resource when it's created

File: c7n/resources/cw.py (hunk 1 above)
Comment (joshuaroot) [reply c7n/resources/cw.py]: ah, that's what i was looking for.

File: tests/test_cwl.py (hunk 2 above)
Comment (joshuaroot) [reply tests/test_cwl.py]: lol, you're right... used to having the resource in play after the `tag` test... forgot i was creating it new here.

**Comment by kapilt** [comment]
this is already handled better by #2364 using resource tagging api

**Comment by joshuaroot** [comment]
then i shall close this one out.

**Pull request closed without merging**